AsyncDisplayKit/ASTableView.mm

```diff
@@ -737,6 +737,7 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
 - (CGFloat)tableView:(UITableView *)tableView heightForRowAtIndexPath:(NSIndexPath *)indexPath
 {
   ASCellNode *node = [_dataController nodeAtIndexPath:indexPath];
+  [node layoutIfNeeded];
```

Comment:
That is removed now as we return the `pendingLayout` if exists or the `calculatedLayout` size for `calculatedSize`